internal/commands/secret.go

```diff
@@ -112,9 +112,13 @@ func (b *commandsBuilder) newConfigCmd() *configCmd {
 
 	removeCmd.Flags().StringVar(&cc.vaultType, "type", "", "vault type")
 	removeCmd.Flags().StringVar(&cc.path, "path", "", "path to secrets")
+	removeCmd.MarkFlagRequired("type")
+	removeCmd.MarkFlagRequired("path")
 
 	setCmd.Flags().StringVar(&cc.vaultType, "type", "", "vault type")
 	setCmd.Flags().StringVar(&cc.filePath, "file", "", "file with sercrets")
+	setCmd.MarkFlagRequired("type")
```

Comment:
Can we also print the usage of the sub command after the error?

Reply:
Also, if talking idealistic way, `ParameterAlreadyExists` error doesn't make sense when we deal with `secrets`. Maybe we could catch the error and use `Secret already exists, please use `--overwrite` or something like that?

(also maybe we should rename `--overwrite` to `--force`?)

Reply:
Yes, by default viper after every error can print a help message, but the error maybe is not be syntax. I see strange this when an error in logic and app print help message. If you want to use this feature with print help message I see the way when we do not send logic errors up and handle them in function command. Thus we do not print help messages when catching a logic error

Reply:
`ParameterAlreadyExists ` is a standard error in AWS SDK

Reply:
So can we catch the Parameter Store error easily? as it's a bit confusing for the user as they don't really know what a "parameter" as they are dealing with "Secrets". If we can't do it easily, let's skip this for now.

Reply:
Yes, we can catch this error - it's easy, and we can also just overwrite secrets by setting the overwrite parameter to true.